source/localizable/tutorial/installing-addons.md

```diff
@@ -18,7 +18,11 @@ Run the following command to install the addon:
 ember install ember-cli-tutorial-style
 ```
 
-Starting the server will incorporate the new CSS and refreshing our browser window will give you this:
+Since Ember addons are regular npm packages, `ember install` installs them in
```

Comment:
Not quite "regular" as there is a bit more to them, such as blueprints.  Maybe phrase it, "Since Ember addons are npm packages, Running `ember install` puts them in the `node_modules` directory..."